### DISCUSSION TITLE
Self Update does not appear as a command.

### DISCUSSION THREAD

**Discussion by cjsage52** [Taxel/PlexTraktSync#1148] [Support] [answered]
I installed via pipx on Windows 10 but the self update command isn't available. I can update through pipx directly but just wondered if this command is supposed to be available.

**Comment by erikt311** [comment]
This was working fine for me until 0.22.4

Now, self-update does not appear to be a valid command. 

However, using this still works:

pipx upgrade plextraktsync

**Reply by cjsage52** [reply]
I use the upgrade command through pipx as well as a work around.

**Comment by glensc** [comment, marked as answer]
you both need to report this as an issue.

and when you do, don't disappear like last user:
- https://github.com/Taxel/PlexTraktSync/issues/1444#issuecomment-1608912664
